qemu/tests/getfd.py

```diff
@@ -52,7 +52,7 @@ def has_fd(pid, filepath):
         path = os.path.join(fdfiles_dir, name)
         fd = os.open(path, os.O_RDWR | os.O_CREAT)
         response = vm.monitor.getfd(fd, name)
-        os.close(fd)
+        vm.monitor.closefd(fd, name)
```

Comment:
Yes, you opened the fd in avocado process and passed it to qemu, so the qemu process had that fd as well. But then qemu's fd would be closed by `vm.monitor.closefd(fd, name)`, thus, I think it should be failed when you wanted to ensure if the fd is presented in qemu.